README.md

```diff
@@ -14,25 +14,35 @@ npm install --save @nearform/quantum
 
 #### With Tailwind
 
-Inclue or extend our colour configuration and add our components to Tailwind's content configuration.
+Inclue or extend our colour configuration and add our components to Tailwind's content configuration. The quantumPlugin is a function that can accept a `node_modules_path` string variable, but is only required if your tailwind configs `content.relative` is set to true.
 
 ```js
 // tailwind.config.js
-import colors from '@nearform/quantum/dist/colors'
+const quantumPlugin = require('@nearform/quantum/plugin')
```

Comment:
why did we switch from import to require? also, I would call the plugin tailwind-plugin to make it clearer what plugin it is